src/winrtble/peripheral.rs

```diff
@@ -101,6 +150,8 @@ impl Peripheral {
             .raw_signal_strength_in_dbm()
             .ok()
             .map(|rssi| rssi as i8);
+
+        self.adapter.emit(CentralEvent::DeviceUpdated(self.address));
```

Comment:
It looks like you send this even if the only change is the service data or manufacturer data, which you already send an event for above, whereas in the Mac OS implementation I think you only send the `ServiceAdvertisement`/`ManufacturerAdvertisement` event.

I suggest that you only send the `DeviceUpdated` event if some other property of the device has changed.